### ISSUE TITLE
Resolve duplicate Lithunian records

### ISSUE THREAD

**Issue by kd-ods** [org-id/register#421]
We currently have two records for Lithuania:

- http://org-id.guide/list/LT-RC

and

- http://org-id.guide/list/LT-PVM

(https://github.com/org-id/register/tree/master/lists/lt)

They point to the same register, though. This looks like a clear case of mistaken duplication. I suggest we resolve this by:

- Keeping the http://org-id.guide/list/LT-RC list since the information there is more detailed and useful.
- Marking LT-PVM as deprecated and editing BOTH records to say that LT-RC should now be used in preference to LT-PVM

**Comment by kd-ods** [comment]
Yup - it looks like LT-PVM needs some work, as you say. Having a look around, PVM numbers look to be searchable here at the State Tax Inspectorate: https://www.vmi.lt/cms/en/informacija-apie-mokesciu-moketojus (Under the company search tab).

This could be a better place to link to for the PVM.

**Comment by siwhitehouse** [comment]
I've raised PR #433 as a response to this issue and asked @kd-ods to review it.
